src/EncryptedModels.ts

```diff
@@ -400,7 +400,7 @@ export class EncryptedCollection {
   public accessLevel: CollectionAccessLevel;
   public stoken: string | null; // FIXME: hack, we shouldn't expose it here...
 
-  public static async create(parentCryptoManager: AccountCryptoManager, collectionTypeName: string, meta: ItemMetadata, content: Uint8Array): Promise<EncryptedCollection> {
+  public static async create<T = {}>(parentCryptoManager: AccountCryptoManager, collectionTypeName: string, meta: ItemMetadata<T>, content: Uint8Array): Promise<EncryptedCollection> {
```

Comment:
Do you really all of the `= {}` part of the generic? I think this is implicit, at least for setters, no?

Reply:
Yes, works without. Thanks changed 👍